docs/reference/ml/ml-shared.asciidoc

```diff
@@ -963,10 +964,11 @@ is `shared`, which generates an index named `.ml-anomalies-shared`.
 end::results-index-name[]
 
 tag::results-retention-days[]
-Advanced configuration option. The number of days for which job results are 
-retained. Once per day at 00:30 (server time), results older than this period
-are deleted from {es}. The default value is null, which means results are
-retained.
+Advanced configuration option. When set this option will prune results older
+than this age in days. Age is calculated as the difference between the latest
```

Comment:
Note sure if newest or latest is correct, bit I think it should be used similarly in both definitions:
```suggestion
Age is calculated relative to the timestamp of the latest bucket result.
```